src/components/CioQuiz/quizLocalReducer.ts

```diff
@@ -92,6 +92,31 @@ export default function quizLocalReducer(
         isQuizCompleted: false,
       };
     }
+    case QuestionTypes.Skip: {
+      const { answers, answerInputs } = state;
+      const newAnswers = [...answers];
+      const lastAnswerInputIndex = answers.length;
+      const currentAnswerInput = Object.values(state.answerInputs)?.[lastAnswerInputIndex];
+      switch (currentAnswerInput.type) {
```

Comment:
The previous line has optional chaining, does that mean `currentAnswerInput` could be null/undefined? If yes, is it possible for currentAnswerInput.type to error out? I just realized it's the same in `Next` so maybe it's a question for @mocca102 as well

Reply:
The optional chaining actually should be removed.

Reply:
It's not doing anything but it's throwing whomever reading the code off

Reply:
Removed [f18aa78](https://github.com/Constructor-io/constructorio-ui-quizzes/pull/132/commits/f18aa78821419475ac9b701bcd44f4f165819fca)